caches.json

```diff
@@ -15,6 +15,14 @@
       "cacheDomain": "cdn.cloudflare.com",
       "updateCacheApiDomainSuffix": "amp.cloudflare.com",
       "thirdPartyFrameDomainSuffix": "cloudflareamp.net"
+    },
+    {
+      "id": "bing",
+      "name": "Bing AMP Cache",
+      "docs": "https://www.bing.com/",
```

Comment:
Do you have a better documentation URL?

Maybe https://www.bing.com/webmaster/help/bing-amp-cache-bc1c884c ?

Reply:
I do. And I thought I had it there. My bad. I will update it

Reply:
Done